src/easyzoom.js

```diff
@@ -73,6 +79,8 @@
         var w1, h1, w2, h2;
         var self = this;
 
+        if (!this.opts.beforeShow.call(this)) return;
```

Comment:
The default value is delegated to `$.noop` which will return `undefined` which will make this return early.